### PR TITLE
Added error handler file

### PR DESCRIPTION
**Pull request by Sable-20**
Added a more extensive error handler and removed old one.

I also added a few comments into mine so you can tell whats going on

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+def convert(time): #makes life easier and looks better overall
+    day = time // (24 * 3600)
+    time = time % (24 * 3600)
+    hour = time // 3600
+    time %= 3600
+    minutes = time // 60
+    time %= 60
+    seconds = time
+    return "%d:%d:%d:%d" % (day, hour, minutes, seconds)
```
Comment (WizzyGeek) [inline manage.py:14]: Use an f-string, as it is faster and much more readable

File: manage.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                message = f'Retry this command after **{x[3]}** seconds!'
+            msg = await ctx.send(message)
+            await sleep(3)
+            await msg.delete()
+        if isinstance(error, commands.MissingRequiredArgument):
+            msg = await ctx.send("**You have made an error.**\n\n{}".format(error.param))
+            await sleep(3) #missing params, can get rid of
+            await msg.delete()
+        if isinstance(error, commands.TooManyArguments):
+            msg = await ctx.send('You have given too many args.\nPlease use the command as directed.')
```
Comment (WizzyGeek) [inline manage.py:46]: You can use a `dict[Exception, str]` to refactor beyond this point to make it shorter, faster and more readable

File: manage.py (hunk 1 above)
Comment (Sable-20) [reply manage.py:14]: I can change but is it actually faster? i didnt know xD, ive been doing C recently and it uses that style

**Review by WizzyGeek — CHANGES_REQUESTED** [review]

File: manage.py (hunk 2 above)
Comment (Sable-20) [reply manage.py]: **done**

File: manage.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        elif isinstance(error, commands.BotMissingAnyRole):
+            msg = await ctx.send('**Woops!**\n\nLooks like i am missing the {} role.'.format(error.missing_role))
+            await sleep(3) #bot missing role
+            await msg.delete()
+        elif isinstance(error, commands.errors.NotOwner):
+            msg = await ctx.send('Only **{}** can use this command.'.format(ctx.guild.owner))
+            await sleep(3) #owner only commmands
+            await msg.delete()
+        else:
+            await ctx.send(switch.get(error, "an unknown error occured"), delete_after=3.0)
```
Comment (WizzyGeek) [inline manage.py:75]: I am sorry, I totally forgot, it is `type(error)`

**Review by WizzyGeek — CHANGES_REQUESTED** [review]

File: manage.py (hunk 3 above)
Comment (Sable-20) [reply manage.py]: dw i got it xD, sorry for missing it i havent used this style of error handler before

**Review by WizzyGeek — APPROVED** [review]

**Comment by WizzyGeek** [comment]
You forgot to load these cogs, move them to the cogs folder so that they get loaded on launch

**Comment by Sable-20** [comment]
> 
> 
> You forgot to load these new cogs, move them to the cogs folder so that they get loaded on launch

did i actually, oops

**Comment by Sable-20** [comment]
> 
> 
> You forgot to load these new cogs, move them to the cogs folder so that they get loaded on launch

fixed

**Review by WizzyGeek — APPROVED** [review]
LGTM 👍🏼

**Review by Nemika-Haj — APPROVED** [review]
Fair enough, merging.

**Pull request merged**